packages/blade/src/components/DatePicker/types.ts

```diff
@@ -145,20 +154,29 @@ type DatePickerProps<Type extends DateSelectionType> = Omit<
      * @example 'date' | { start: 'start-date', end: 'end-date' }
      */
     name?: Type extends 'single' ? string : { start: string; end?: string };
+    helpText?: Type extends 'single' ? string : { start: string; end?: string };
+    errorText?: Type extends 'single' ? string : { start: string; end?: string };
+    successText?: Type extends 'single' ? string : { start: string; end?: string };
     labelPosition?: BaseInputProps['labelPosition'];
   };
 
 type DatePickerRangeInputProps = {
   selectionType: 'range';
   label?: { start: string; end?: string };
   name?: { start: string; end: string };
+  successText?: { start: string; end?: string };
+  errorText?: { start: string; end?: string };
+  helpText?: { start: string; end?: string };
   date: [Date, Date];
 };
 
 type DatePickerSingleInputProps = {
   selectionType: 'single';
   label?: string;
   name?: string;
+  successText?: string;
+  errorText?: string;
+  helpText?: { start: string; end: string };
```

Comment:
Shouldn't this be a string in case of the single input?

```suggestion
  helpText?: string;
```

Reply:
nice find.